dynamic.js

```diff
@@ -123,11 +147,25 @@ module.exports = function (fastify, opts, next) {
         }
 
         if (schema.body) {
-          const consumesAllFormOnly =
-              consumesFormOnly(schema) || consumesFormOnly(swaggerObject)
-          consumesAllFormOnly
-            ? getFormParams(parameters, schema.body)
-            : getBodyParams(parameters, schema.body)
+          if (isOAS3) {
+            const consumes = schema.consumes || swaggerObject.consumes
+            swaggerMethod.requestBody = getRequestBodyParams(schema.body, consumes)
+          } else {
+            const consumesAllFormOnly = consumesFormOnly(schema) || consumesFormOnly(swaggerObject)
+            consumesAllFormOnly
+              ? getFormParams(parameters, schema.body)
+              : getBodyParams(parameters, schema.body)
+          }
+        }
+
+        if (schema.requestBody) {
+          const content = schema.requestBody.content
+          parameters.push(content)
+          // const consumesAllFormOnly =
+          //     consumesFormOnly(schema) || consumesFormOnly(swaggerObject)
+          // consumesAllFormOnly
+          //   ? getFormParams(parameters, schema.body, !!swaggerObject.openapi)
+          //   : getBodyParams(parameters, schema.body, !!swaggerObject.openapi)
```

Comment:
why is this block commented out?

Reply:
This will be removed. I've made correct implementation within body processing.